src/pybind/mgr/call_home_agent/config.src

```diff
@@ -0,0 +1,47 @@
+import os
+import sys
+
+from cryptography.fernet import Fernet, InvalidToken
+
+defaultKeyFile = ''
+defaultKey = b''
+settings = {
+    'api_key': b'',
+    'private_key': b''
+}
```

Comment:
How's this file meant to be used? As per my understanding, this is not working and the plan is to keep it this way for upstream? So how's this supposed to be tested upstream?

Reply:
Do we need to test this upstream?

Reply:
If we plan to contribute this upstream (which I'm still hesitant about), some automated testing should be in place (at least a static checker/pylint/mypy, and some unit tests). Otherwise, we are imposing ourselves the (non-trivial) extra step of going through the upstream release cycle for getting little benefit from that (no CI, no packaging, no integration testing, ...). This would become kinda dead code hanging from a community repo (which in some sense will probably be).